tests/openQA/dashboard.pm

```diff
@@ -9,11 +9,6 @@ sub run {
     ensure_unlocked_desktop();
     x11_start_program("firefox http://localhost", 60, { valid => 1 } );
     # starting from git might take a bit longer to get and generated assets
-    # workaround for poo#19798, basically doubles the timeout
-    if ((check_screen 'openqa-dashboard', 180) == undef) {
-        record_soft_failure 'ff took to long to start';
```

Comment:
Please leave the record_soft_failure, it hasn't happened in this verification run, but it can happened again in the future. The only thing you have to add, it the bug reference (link to bugzilla).

We augmented the timeout to make the test green as workaround and complete other steps of the test. But that firefox take so much time to start is still for me a bug.